### ISSUE TITLE
The Flow 0.53 node.js lib def updates are missing

### ISSUE THREAD

**Issue by lll000111** [facebook/flow#4662]
Summary: The commit (link: see below) that fixed this and that's supposed to be in 0.53(.1) doesn't seem to be there,

### Flow version

Windows

```
$ flow version
Flow, a static type checker for JavaScript, version 0.53.1
```

### Code

```javascript
// @flow
'use strict';

const fs = require('fs');

fs.mkdir('directory', (err: ?ErrnoError): void => console.log(err));

fs.open('filename', 'r', (err: ?ErrnoError, fd: number): void => console.log(err));
```

### Flow Try

[Link to flow.org/try example](https://flow.org/try/#0PTAEAEDMBsHsHcBQByArgZwKanQFwE4CWAxrsgNyKLGwB2eok6oAvKPpgI6qEcAUyJsgCUlREwB0AWwDWAE14CFHUrHwBPZABpQfTPnwAuUAH4AogdqwL+NcOMA3WITmsAfKBr1Y0TBLgA5noGwqJUkrAADpi0ApCEvrQAhlKY2qDI+OnBRqY2VjZqOpByxrSoUgBG+vagTi7unnToPn6BOaGUQA)

### Flow output

```
Error: test.js:6
  6: fs.mkdir('directory', (err: ?ErrnoError): void => console.log(err));
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ function. This type is incompatible with the expected param type of
776:   declare function mkdir(path: string, mode?: number, callback?: (err: ?ErrnoError) => void): void;
                                                   ^^^^^^ number. See lib: ../../../AppData/Local/Temp/flow/flowlib_2dc3cdd6/node.js:776

Error: test.js:8
  8: fs.open('filename', 'r', (err: ?ErrnoError, fd: number): void => console.log(err));
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ function. This type is incompatible with the expected param type of
784:   declare function open(path: string | Buffer, flags: string | number, mode?: number, callback?: (err: ?ErrnoError, fd: number) => void): void;
                                                                                   ^^^^^^ number. See lib: ../../../AppData/Local/Temp/flow/flowlib_2dc3cdd6/node.js:784


Found 2 errors
```

### Flow commit

https://github.com/facebook/flow/commit/266c87a71fdfb16109e7cc039c1810cd56455501

I don't see `mkdir` patched in that commit,  [but `open` was patched](https://github.com/facebook/flow/commit/266c87a71fdfb16109e7cc039c1810cd56455501#diff-0e57e9bd985a92cc0d6572193082ed61R795).

### Local lib file excerpt

Using the error message above, here is the section from that local lib.def. file around line 784 (`open`):

```
...
declare function readdir(path: string, callback?: (err: ?ErrnoError, files: Array<string>) => void): void;
declare function readdirSync(path: string): Array<string>;
declare function close(fd: number, callback?: (err: ?ErrnoError) => void): void;
declare function closeSync(fd: number): void;
declare function open(path: string | Buffer, flags: string | number, mode?: number, callback?: (err: ?ErrnoError, fd: number) => void): void;
declare function openSync(path: string | Buffer, flags: string | number, mode?: number): number;
declare function utimes(path: string, atime: number, mtime: number, callback?: (err: ?ErrnoError) => void): void;
declare function utimesSync(path: string, atime: number, mtime: number): void;
declare function futimes(fd: number, atime: number, mtime: number, callback?: (err: ?ErrnoError) => void): void;
declare function futimesSync(fd: number, atime: number, mtime: number): void;
declare function fsync(fd: number, callback?: (err: ?ErrnoError) => void): void;
declare function fsyncSync(fd: number): void;
...
```

<br>

That is different from what the linked commit shows.  Apparently that commit is not actually in the 0.53 release?

From [the 0.53 release notes](https://github.com/facebook/flow/releases/tag/v0.53.0):

> This release also includes lots of improvements to core type definitions.

Does it?

**Comment by asolove** [comment]
@calebmer needs label "Library definitions".

**Comment by calebmer** [comment]
Unfortunately this PR did not make it into the Flow 0.53 branch cut. The branch for 0.53 was cut here: https://github.com/facebook/flow/commit/ac4e7e37851406a518c5f551c00996e20ace092e and we patched some commits on top of that. For the branch see: https://github.com/facebook/flow/tree/v0.53

These changes will be included in the 0.54 release 🎉
